admin/proc-rhdh-deploy-eks-using-operator.adoc

```diff
@@ -287,15 +287,15 @@ $ sed -i "s/backstage-controller-manager/rhdh-controller-manager/g" rhdh-operato
 .. Replace the default images with the images that are pulled from the Red Hat Ecosystem:
 +
 --
-[source]
+[source,console,subs="attributes+"]
```

Comment:
What is it that you are trying to achieve with this listing block? I'm not sure if this syntax is valid. Might be safer to revert to `[source]` since this change is not required and might break the build.

Reply:
this is required to turn `{product-version}` into `1.2`

Reply:
```suggestion
[source,subs="attributes+"]
```